options/locale/locale_en-US.ini

```diff
@@ -679,6 +680,14 @@ saved_successfully = Your settings were saved successfully.
 privacy = Privacy
 keep_activity_private = Hide Activity from profile page
 keep_activity_private_popup = Makes the activity visible only for you and the admins
+heatmap_visibility_popup = Specify which activities are visible in heatmap
+heatmap_visibility = Visible activities in heatmap
+heatmap_visibility.public_popup = Only activies from repositories which can be visited from the viewer are in the heatmap
+heatmap_visibility.public = Public
+heatmap_visibility.all_popup = Only activies from repositories which can be visited from the viewer are in the heatmap
+heatmap_visibility.all = All
+heatmap_visibility.none_popup = Don't show the heatmap
+heatmap_visibility.none = None
```

Comment:
Would move `All`, `None` and `Public` to ini top-level, if they don't exist there already.